test/e2e/specs/wp-likes-spec.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import { By } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import * as driverManager from '../lib/driver-manager';
+import * as dataHelper from '../lib/data-helper';
+import * as driverHelper from '../lib/driver-helper';
+import LoginFlow from '../lib/flows/login-flow';
+import PostAreaComponent from '../lib/pages/frontend/post-area-component';
+import CommentsAreaComponent from '../lib/pages/frontend/comments-area-component';
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+import ProfilePage from '../lib/pages/profile-page';
+import NavBarComponent from '../lib/components/nav-bar-component.js';
+import LoggedOutMasterbarComponent from '../lib/components/logged-out-masterbar-component';
+import AsyncBaseContainer from '../lib/async-base-container';
+
+const host = dataHelper.getJetpackHost();
+const screenSize = driverManager.currentScreenSize();
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const mochaTimeoutMS = config.get( 'mochaTimeoutMS' );
+const blogPostTitle = dataHelper.randomPhrase();
+const blogPostQuote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ screenSize })`, function () {
+	let driver;
+	let postUrl;
+	this.timeout( mochaTimeoutMS );
+
+	before( 'Start browser', async function () {
+		this.timeout( startBrowserTimeoutMS );
+		driver = await driverManager.startBrowser();
+	} );
+
+	describe( 'Like posts and comments @parallel', function () {
+		step( 'Login, create a new post and view it', async function () {
+			this.loginFlow = new LoginFlow( driver, 'gutenbergSimpleSiteUser' );
+			await this.loginFlow.loginAndStartNewPost( null, true );
```

Comment:
It doesn't look like we need to assign the login flow to an instance variable (I don't think it would persist anyway?). Anyway, I think it can be just:
```suggestion
			await new LoginFlow( driver, 'gutenbergSimpleSiteUser' ).loginAndStartNewPost( null, true );
```